skiko/src/jvmMain/cpp/common/StdVectorDecoder.cc

```diff
@@ -0,0 +1,20 @@
+#include <jni.h>
+#include "interop.hh"
+
+extern "C" JNIEXPORT jint JNICALL Java_org_jetbrains_skia_StdVectorDecoderKt_StdVectorDecoder_1nGetArraySize
+    (JNIEnv* env, jclass jclass, jlong ptr) {
+        std::vector<jlong>* vect = reinterpret_cast<std::vector<jlong> *>(ptr);
+        return static_cast<jint>(vect->size());
+    }
+
+extern "C" JNIEXPORT void JNICALL Java_org_jetbrains_skia_StdVectorDecoderKt_StdVectorDecoder_1nDisposeArray
+    (JNIEnv* env, jclass jclass, jlong ptr) {
+        std::vector<jlong>* vect = reinterpret_cast<std::vector<jlong> *>(ptr);
```

Comment:
As @VeselovAlex mentioned, we likely shall have destructor for vector elements either explicitly I.e. stored in first vector element, or implicitly via type parameter.